fe/src/main/java/org/apache/doris/common/util/RuntimeProfile.java

```diff
@@ -39,26 +39,32 @@
 import java.util.Map;
 import java.util.Set;
 
+/**
+ * It is accessed by two kinds of threads, one is thread which create
+ * RuntimeProfile, namely query thread, the other is thread which call
+ * {@link org.apache.doris.common.proc.CurrentQueryInfoProvider}.
```

Comment:
It is accessed by two kinds of thread. One is to create
this RuntimeProfile, named 'query thread', the other is to call this profile.

Reply:
What I want to emphasize is that the thread which call CurrentQueryInfoProvider in 'show proc' . The last words seemed redundant. i will fix the grammar problem.